vars/functionalTestPostV2.groovy

```diff
@@ -35,6 +34,9 @@ void call(Map config = [:]) {
 
     junit(testResults: junit_results)
 
+    // Save these temporarily
+    archiveArtifacts(artifacts: junit_results)
+
```

Comment:
So you are going to remove this before landing?  I will -1 here assuming that is a yes to prevent an accidental landing of it.